dbt/include/snowflake/macros/materializations/merge.sql

```diff
@@ -58,10 +58,10 @@
 
     {#-- Add additional incremental_predicates to filter for batch --#}
     {% if model.config.get("__dbt_internal_microbatch_event_time_start") -%}
-        {% do incremental_predicates.append("DBT_INTERNAL_TARGET." ~ model.config.event_time ~ " >= TIMESTAMP '" ~ model.config.__dbt_internal_microbatch_event_time_start ~ "'") %}
+        {% do incremental_predicates.append("DBT_INTERNAL_TARGET." ~ model.config.event_time ~ " >= to_timestamp_tz('" ~ model.config.__dbt_internal_microbatch_event_time_start ~ "')") %}
```

Comment:
Interestingly, just using the raw timestamp (that includes UTC offset, which `model.config.__dbt_internal_microbatch_event_time_start` does) works well. E.g. just removing `TIMESTAMP` would have been enough here. However, the `to_timestamp_tz` piece for more intentional so I've opted for that.

Reply:
I like how using `TIMESTAMP` was causing the issue 🫠 I like th emove to using `to_timestamp_tz` instead of just removing `TIMESTAMP` 👍